template/terraform/main.tf

```diff
@@ -1,9 +1,25 @@
+provider "vault" {
+  address = "http://127.0.0.1:8200"
+  token = "master"
+}
+
 provider "nomad" {
   address = "http://127.0.0.1:4646"
+  // The following line is needed if ACLs are enabled in Nomad
```

Comment:
We probably need make research about conditional rendering.

Reply:
Agreed, now that you mention it I have an idea, so I'll tinker with that a little before I call this done

Reply:
Forgot to say, but same as above, this should be updated, but will create an issue and bundle it in another PR.